common/authorization/default_jwt_claim_mapper.go

```diff
@@ -125,7 +125,18 @@ func (a *defaultJWTClaimMapper) extractPermissions(permissions []interface{}, cl
 }
 
 func parseJWT(tokenString string, keyProvider TokenKeyProvider) (jwt.MapClaims, error) {
-	token, err := jwt.Parse(tokenString, func(token *jwt.Token) (interface{}, error) {
+	return parseJWTWithAudience(tokenString, keyProvider, "")
+}
+
+func parseJWTWithAudience(tokenString string, keyProvider TokenKeyProvider, audience string) (jwt.MapClaims, error) {
+
+	var parser *jwt.Parser
+	if audience == "" {
```

Comment:
[Nit] Can we do strings.Trim(audience) == "" just to make sure empty space is also treated as no audience?

Reply:
Done.